### DISCUSSION TITLE
Presets for audio and subtitle passthrough

### DISCUSSION THREAD

**Discussion by raadhey** [jlesage/docker-handbrake#299] [Q&A]
I'm using the docker container on Ubuntu server OS installed on an Intel i5-7500T machine. I am trying to automate the conversion via the /watch directory. 
I was able to create a preset from the webgui to select H265 encode. 
I would also like to ensure audio passthrough and (if possible) subtitle passthrough. 
Manually I can set it the passthrough for a file/ conversion job, but when I try to save it as a preset I see that it has no effect. 

So my Question is whether it is actually possible to set audio and subtitle passthrough as a preset?
My intention is to convert to H265 to reduce file size and keep CPU utilization low (QSV for the video encoding) by not doing audio decode-encode. 

Please let me know if I have missed these steps from previous discussions. I have tried to search extensively and also looked through reddit and other general searching around. 

Thanks.

**Comment by jlesage** [comment]
Under Audio and Subtitles tabs, at the left you can select "Track Selection".  This is where the magic happens.
For Audio, see https://github.com/jlesage/docker-handbrake/issues/222#issuecomment-1167340789.
For Subtitles, this might help: https://www.reddit.com/r/handbrake/comments/nlm4gl/passthrough_all_subtitles_in_mkv/
